eoscostarica/eos-rate
# refactor(webapp): trim profile page code down, add redirect to not-found

This quite a refactor, please be thorough when looking at the code changes.

### Steps to test
1. Go into block producers page.
1. Select a block-producer profile page.
1. Look at the redux state, it shouldn't have any change related to loading the profile page.
1. Profile page should look exactly as production does.
1. Try to navigate to a non existent account like `/block-producers/eostest`.
1. You should get redirected to the `Not found` page.
1. Hitting back, should return you to the last valid address.
